spec/models/review_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+
+  context 'reviews' do
+    let!(:review) do
+      FactoryGirl.create(:review)
+    end
+
+    it { should belong_to(:tattoo) }
+    it { should belong_to(:user) }
+
+    it { should validate_presence_of(:user_id) }
+    it { should validate_presence_of(:tattoo_id) }
+    it { should validate_presence_of(:rating) }
+    it { should validate_numericality_of(:rating) }
+    it { should validate_inclusion_of(:rating).
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.